programs/client/Client.cpp

```diff
@@ -963,9 +963,7 @@ bool Client::processWithFuzzing(const String & full_query)
 void Client::printHelpMessage(const OptionsDescription & options_description)
 {
     std::cout << options_description.main_description.value() << "\n";
-    std::cout << options_description.external_description.value() << "\n";
-    std::cout << options_description.hosts_and_ports_description.value() << "\n";
-    std::cout << "In addition, --param_name=value can be specified for substitution of parameters for parametrized queries.\n";
+    std::cout << "In addition, if you want to see more hints you can go to https://clickhouse.com/docs.\n";
```

Comment:
If I run `./clickhouse-client --help`, I am still flooded by hundreds of options and printing "if you want to see more hints" appears a bit ironic 😄  Intended?